[2022-01-28 12:37:10,303] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-01-28T12:37:05.738838+00:00 [queued]>
[2022-01-28 12:37:10,337] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-01-28T12:37:05.738838+00:00 [queued]>
[2022-01-28 12:37:10,338] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-28 12:37:10,340] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-28 12:37:10,342] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-28 12:37:10,373] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-01-28 12:37:05.738838+00:00
[2022-01-28 12:37:10,392] {standard_task_runner.py:52} INFO - Started process 5980 to run task
[2022-01-28 12:37:10,400] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'manual__2022-01-28T12:37:05.738838+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dags.py', '--cfg-path', '/tmp/tmp0dll7r41', '--error-file', '/tmp/tmp565js0hp']
[2022-01-28 12:37:10,404] {standard_task_runner.py:77} INFO - Job 32: Subtask download_dataset_task
[2022-01-28 12:37:10,577] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-01-28T12:37:05.738838+00:00 [running]> on host 6aa22590a839
[2022-01-28 12:37:10,815] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-28T12:37:05.738838+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-28T12:37:05.738838+00:00
[2022-01-28 12:37:10,823] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-28 12:37:10,834] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv > /opt/***/yellow_tripdata_2021-01.csv']
[2022-01-28 12:37:10,874] {subprocess.py:85} INFO - Output:
[2022-01-28 12:37:50,863] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-28 12:37:50,956] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20220128T123705, start_date=20220128T123710, end_date=20220128T123750
[2022-01-28 12:37:51,071] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-28 12:37:51,184] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
